openreview/conference/matching.py

```
@@ -251,6 +251,92 @@ def _build_conflicts(self, submissions, user_profiles, get_profile_info):
             return self._build_profile_conflicts(ac_profiles, user_profiles)
         return self._build_note_conflicts(submissions, user_profiles, get_profile_info)
 
+    def append_note_conflicts(self, user_profile, build_conflicts=None):
+        '''
+        Create conflict edges between the given Notes and a single profile
+        '''
+
+        # Re-setup information that would have been initialized in setup()
+        submissions = list(openreview.tools.iterget_notes(
+            self.conference.client,
+            invitation=self.conference.get_blind_submission_id(),
+            details='original'))
+        get_profile_info = openreview.tools.get_neurips_profile_info if build_conflicts == 'neurips' else openreview.tools.get_profile_info
+        self.match_group = openreview.tools.replace_members_with_ids(self.client, self.match_group)
+        if not all(['~' in member for member in self.match_group.members]):
+            print(
+                'WARNING: not all reviewers have been converted to profile IDs.',
+                'Members without profiles will not have metadata created.')
+        
+        # Adapt single profile to multi-profile code
+        user_profiles = [user_profile]
+        user_profiles = _get_profiles(self.client, user_profiles, with_publications=build_conflicts)
+        user_profiles_info = [get_profile_info(p) for p in user_profiles]
+
+        # Fetch conflict invitation
+        try:
+            invitation = self.client.get_invitation(self.conference.get_conflict_score_id(self.match_group.id))
+        except:
+            print('Conflict invitation not found')
```

can you raise an OpenReview Exception here?